### PR TITLE
NOTIF-140 Enable RBAC user queries

### PR DESCRIPTION
**Pull request by gwenneg**

### CODE REVIEW

**Comment by gwenneg** [comment]
The test failure is unrelated (aggregator). Working on it.

**Comment by gwenneg** [comment]
Test disabled, we'll fix it later.

**Comment by gwenneg** [comment]
This is printed at application startup:
`2021-09-02T09:47:07.3476348Z 2021-09-02 09:47:07,347 INFO  [com.red.clo.not.pro.ema.RecipientResolver] (main) RBAC user queries are enabled`

File: aggregator/src/test/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJobTest.java [hunk 1]
```diff
@@ -37,6 +38,7 @@ void init() {
     }
 
     @Test
+    @Disabled // TODO Test disabled because it blocked a prod deployment.
```
Comment (gwenneg) [inline aggregator/src/test/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJobTest.java:41]: @bennetelli This is the failing test I mentionned this morning on Slack.

Comment (bennetelli) [reply aggregator/src/test/java/com/redhat/cloud/notifications/processors/email/aggregators/DailyEmailAggregationJobTest.java:41]: I fixed this in #474 by adding

    @BeforeEach
    void setUp() {
        helpers.purgeAggregations();
        helpers.purgeEmailSubscriptions();
    }

    @AfterEach
    void tearDown() {
        helpers.purgeAggregations();
        helpers.purgeEmailSubscriptions();
    }

**Review by theute — APPROVED** [review]

**Pull request merged**